packages/account/src/Components/api-token/api-token-card.tsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-import { Field, FieldProps } from 'formik';
+import { Field, FieldProps, useFormikContext } from 'formik';
 import { CompositeCheckbox } from '@deriv/components';
 
 type TApiTokenCard = {
     description: string;
     display_name: string;
     name: string;
     value: boolean;
-    setFieldValue: (name: string, value: boolean) => void;
 };
 
-const ApiTokenCard = ({
-    name,
-    value,
-    display_name,
-    description,
-    setFieldValue,
-    children,
-}: React.PropsWithChildren<TApiTokenCard>) => {
+const ApiTokenCard = ({ name, value, display_name, description, children }: React.PropsWithChildren<TApiTokenCard>) => {
+    const { setFieldValue } = useFormikContext();
     return (
         <Field name={name}>
             {({ field }: FieldProps<string | boolean>) => {
```

Comment:
```suggestion
            {({ field, form: { setFIeldProps } }: FieldProps<string | boolean>) => {
```

Reply:
If it's a checkbox only, can we investigate more on only using boolean type here?

Reply:
you mean form:{setFieldValue} ?

Reply:
My bad. Updated the suggestion